applications/StructuralMechanicsApplication/tests/test_patch_test_membrane.py

```diff
@@ -361,6 +361,155 @@ def test_membrane_wrinkling_law(self):
         self.assertAlmostEqual(mp.Nodes[3].GetSolutionStepValue(KratosMultiphysics.DISPLACEMENT_X), 0.15072065295319598,4)
 
 
+    def test_membrane_cauchy_stress_and_local_axis(self):
+        self.skipTestIfApplicationsNotAvailable("ConstitutiveLawsApplication")
```

Comment:
why?
Which law do you use?

Reply:
ah yes this is a remainder from an old test. I'll delete it